firebase-perf/src/test/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidatorTest.java

```diff
@@ -159,6 +159,14 @@ public void testInvalidCounterSubtrace() {
     assertThat(new FirebasePerfTraceValidator(trace.build()).isValidPerfMetric()).isFalse();
   }
 
+  @Test
+  public void testNonPositiveScreenTraceTotalFrames() {
```

Comment:
Best practice for naming the unit tests: go/unit-testing-practices#naming

Can you rename this test to follow that convention. The other parts of the code can be refactored later. We can make sure to follow the convention for any code we add/change.

Reply:
done